boards/st/stm32u083c_dk/arduino_r3_connector.dtsi

```diff
@@ -36,3 +36,4 @@
 };
 
 arduino_serial: &usart2 {};
+arduino_i2c: &i2c1 {};
```

Comment:
Please also add `arduino_i2c` to `supported:` section of stm32u083c_dk.yaml.yaml file.